add: note to direct users to modifiyng frontmatter with remark (#3988)
withastro/docs: src/content/docs/en/guides/markdown-content.mdx

### Modifying frontmatter programmatically

:::note

> If you are using [content collections](/en/guides/content-collections/), you will instead want to read ["Modifying Frontmatter with Remark"](/en/guides/content-collections/#modifying-frontmatter-with-remark)

Reviewer: ```suggestion
If you are using [content collections](/en/guides/content-collections/), please see ["Modifying Frontmatter with Remark"](/en/guides/content-collections/#modifying-frontmatter-with-remark).
```
A little more direct!
